python-avd/pyavd/_eos_cli_config_gen/j2templates/documentation/traffic-policies.j2

```diff
@@ -55,66 +51,64 @@
+{%                     set row.dst_port = [] %}
+{%                     set row.src_field = [] %}
+{%                     set row.dst_field = [] %}
 {%                     if match.protocols is arista.avd.defined %}
 {%                         set row.protocols = match.protocols | map(attribute='protocol') | join('<br/>') %}
-{%                     else %}
-{%                         set row.protocols = "ANY" %}
-{%                     endif %}
-{#                     -- Source Port #}
-{%                     set row.src_port = "" %}
-{%                     if match.protocols is arista.avd.defined %}
+{#                         -- Source/Destination Port #}
 {%                         for protocol in match.protocols %}
-{%                             if protocol.src_field is arista.avd.defined %}
-{%                                 set row.src_port = protocol.src_field %}
-{%                             elif protocol.src_port is arista.avd.defined %}
-{%                                 set row.src_port = protocol.src_port %}
+{%                             if protocol.protocol | lower in ['tcp','udp'] %}
+{%                                 do row.src_port.append(protocol.src_port | arista.avd.default("all")) %}
+{%                                 do row.src_field.append(protocol.src_field | arista.avd.default("-")) %}
+{%                                 do row.dst_port.append(protocol.dst_port | arista.avd.default("all")) %}
+{%                                 do row.dst_field.append(protocol.dst_field | arista.avd.default("-")) %}
+{%                             else %}
+{%                                 do row.src_port.append("-") %}
+{%                                 do row.src_field.append("-") %}
+{%                                 do row.dst_port.append("-") %}
+{%                                 do row.dst_field.append("-") %}
 {%                             endif %}
 {%                         endfor %}
-{%                         if row.src_port == "" %}
-{%                             set row.src_port = "ANY" %}
-{%                         endif %}
-{%                     endif %}
-{#                     -- Destination Port #}
-{%                     set row.dst_port = "" %}
-{%                     if match.protocols is arista.avd.defined %}
-{%                         for protocol in match.protocols %}
-{%                             if protocol.dst_field is arista.avd.defined %}
-{%                                 set row.dst_port = protocol.dst_field %}
-{%                             elif protocol.dst_port is arista.avd.defined %}
-{%                                 set row.dst_port = protocol.dst_port %}
-{%                             endif %}
-{%                         endfor %}
-{%                         if row.dst_port == "" %}
-{%                             set row.dst_port = "ANY" %}
-{%                         endif %}
+{%                         set row.src_port = row.src_port | join('<br/>') %}
```

Comment:
Please join on comma+space instead <br/> in all these fields. Since we do not attempt to line up between the columns, but just present multiple values, we can let the browser wrap as needed by using `| join(', ')`

Reply:
I misunderstood the layout of each row. Keep the <br>